R/version.R

```diff
@@ -152,3 +151,8 @@ use_c_version <- function(ver) {
 
   write_utf8(version_path, lines)
 }
+
+is_dev_version <- function(version) {
```

Comment:
```suggestion
is_dev_version <- function(version = proj_version()) {
```

Maybe sensible/handy?